static/schemas/2.0/workflowGraphs.2.0.json

```diff
@@ -0,0 +1,59 @@
+{
+    "title": "workflowGraphs.2.0",
+    "definitions": {
+        "tasks": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "oneOf": [
+                    {"required": ["taskName"]},
+                    {"required": ["taskDefinition"]}
```

Comment:
@yyscamper ```label``` isn't listed as a required field in the documentation

http://rackhd.readthedocs.io/en/latest/rackhd/graphs.html?highlight=workflow%20graphs

also, it the workflow graph only has one item then there's no need for a label, correct?